### PR TITLE
Define security api-collections and security-connectors commands

### PR DESCRIPTION
**Pull request by ivadim**
This PR adds commands for security api-collections and security-connectors commands

https://github.com/Azure/azure-cli/pull/28285

### CODE REVIEW

File: Commands/security/api-collection/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# [Group] _security api-collection_
+
+Manage Azure API Management API connections to Microsoft Defender for APIs.
```
Comment (aditdalvi) [inline Commands/security/api-collection/readme.md:3]: "Discover, manage, and view security insights for API collections"

File: Commands/security/api-collection/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Commands
+
+- [create](/Commands/security/api-collection/_create.md)
```
Comment (aditdalvi) [inline Commands/security/api-collection/readme.md:7]: Can we use a different command than "create", like "onboard-api-mgmt-api"? "create" is too generic and will not account for future scenarios where we may want to onboard apiCollections from other resources like App Services or K8s.

File: Commands/security/api-collection/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+: Onboard an Azure API Management API to Microsoft Defender for APIs. The system will start monitoring the operations within the Azure Management API for intrusive behaviors and provide alerts for attacks that have been detected.
+
+- [delete](/Commands/security/api-collection/_delete.md)
```
Comment (aditdalvi) [inline Commands/security/api-collection/readme.md:10]: Can we use a different command than "delete", like "offboard-api-mgmt-api"? "delete" is too generic and will not account for future scenarios where we may want to offboard apiCollections from other resources like App Services or K8s.

Comment (aditdalvi) [reply Commands/security/api-collection/readme.md:10]: Or even just "offboard"

**Review by aditdalvi — CHANGES_REQUESTED** [review]

File: Commands/security/readme.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# [Group] _security_
+
+Manage your security posture with Microsoft Defender for Cloud.
+
+## Subgroups
+
+- [api-collection](/Commands/security/api-collection/readme.md)
+: Manage Azure API Management API connections to Microsoft Defender for APIs.
```
[the remark this answers is not in the document]
Comment (ivadim) [reply Commands/security/readme.md]: Updated

File: Commands/security/api-collection/readme.md (hunk 1 above)
Comment (ivadim) [reply Commands/security/api-collection/readme.md]: Updated

**Review by aditdalvi — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Pull request merged**